src/DataFrames.jl

```diff
@@ -16,6 +16,8 @@ using Reexport
 using GZip
 using SortingAlgorithms
 
+using FileIO  # remove after read_rda deprecation period
```

Comment:
Yeah. I think the dependency should be on RData and the deprecation should use `RData.load` (which I also had before and changed). The deprecation should probably also just be simplified to, as Milan suggested, `@deprecate read_rda(args...) RData.load(args...)`.